### ISSUE TITLE
Learning rate for BatchNorm layers 

### ISSUE THREAD

**Issue by sjbz1926** [YudeWang/deeplabv3plus-pytorch#23]
Hello, thanks for the great work!
I noticed the function “get_param” and it seems not to set the learning rate of Bn layers . Do you think it’s not necessary even harmful to set bn layers learnable? Or somehow I misunderstood.
Thanks!

**Comment by YudeWang** [comment]
@sjbz1926 
The model is pretrained on ImageNet, the training period on VOC can be regard as finetune. Therefore I keep the weight and bias of BN out of learn. 

You can try to include them in "get_param" function. I am curious about the result!
